builtin/providers/aws/structure_test.go

```diff
@@ -256,7 +279,7 @@ func TestExpandIPPerms_nonVPC(t *testing.T) {
 					GroupName: aws.String("sg-22222"),
 				},
 				&ec2.UserIdGroupPair{
-					GroupName: aws.String("sg-22222"),
+					GroupName: aws.String("sg-11111"),
```

Comment:
These are typo fixes. The original tests never checked the second IP Pair returned, but now they do. As such, I'm sure this typo was revealed (you can't add the same security group twice)